index.js

```diff
@@ -307,7 +337,11 @@ function Physics (mcData, world) {
       applyHeading(entity, strafe, forward, acceleration)
       moveEntity(entity, world, vel.x, vel.y, vel.z)
       vel.y *= inertia
-      vel.y -= 0.02
+      if (supportFeature('independentWaterGravity')) {
```

Comment:
You can store the water gravity in a variable at initialization, to avoid the ifs at runtime.